pyuntl/untldoc.py

```diff
@@ -598,7 +598,7 @@ def retrieve_vocab():
     """
     url = VOCABULARIES_URL.replace('all', 'all-verbose')
     try:
-        return eval(urllib2.urlopen(url).read())
+        return urllib.request.urlopen(url).read()
```

Comment:
This is not an equivalent change. We need to decode the JSON response into a Python object.

Reply:
Update the code with the suggested changes to decode the JSON response.

Reply:
Should we have the JSON being decoded here in the try block, rather than outside it? That was the behavior before the upgrade, so we might want to retain that.